config/flags_payments.go

```diff
@@ -65,16 +65,28 @@ var (
 		Name:  "payments.disable",
 		Usage: "Disables payments and moves to a backwards compatible legacy mode",
 	}
-	// FlagPaymentsConsumerUpperPriceBound sets the upper price bound to a set value.
-	FlagPaymentsConsumerUpperPriceBound = cli.Uint64Flag{
-		Name:  "payments.consumer.price.upper.bound",
-		Usage: "Sets the maximum price of the service. All proposals with a price above this bound will be filtered out and not visible.",
-		Value: 1000000,
+	// FlagPaymentsConsumerPricePerMinuteUpperBound sets the upper price bound per minute to a set value.
+	FlagPaymentsConsumerPricePerMinuteUpperBound = cli.Uint64Flag{
+		Name:  "payments.consumer.price.perminute.upper.bound",
```

Comment:
No need to separate every word with a dot. It only makes sense if you are grouping something. Instead of 6 level hierarchy, it could be smth like:

```suggestion
		Name:  "payments.consumer.price-perminute-max,
```
(also replaced upper bound by _max_)